DependencyInjection/Configuration.php

```diff
@@ -36,8 +36,7 @@ public function getConfigTreeBuilder()
                                 ->prototype('array')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('host')->defaultValue('localhost')->end()
-                                        ->scalarNode('port')->defaultValue(61613)->end()
+                                        ->scalarNode('uri')->defaultValue('localhost:61613')->end()
```

Comment:
this is not a necessary BC

Reply:
I think `uri` is correct for greater similarity with centraldesktop/php-stomp (https://github.com/centraldesktop/php-stomp/blob/45a2fa09c6a70fb3c82c8964a3f233d59b2c4205/src/CentralDesktop/Stomp/ConnectionFactory/Simple.php#L8)

Reply:
ok